src/lib/viewers/doc/DocFindBar.js

```diff
@@ -401,6 +401,19 @@ class DocFindBar extends EventEmitter {
         this.opened = false;
         this.bar.classList.add(CLASS_HIDDEN);
     }
+
+    /**
+     * Toggles the Find Bar open and closed
+     *
+     * @return {void}
+     */
+    toggle() {
+        if (!this.opened) {
```

Comment:
We usually want to make the main condition positive, so `if (this.opened)` in this case.